content/java/concepts/math-methods/terms/abs/abs.md

```diff
@@ -11,13 +11,13 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `Math.abs()` method returns the absolute value of the argument.
+The **`Math.abs()`** method returns the absolute value of the argument.
 
 An absolute value is the non-negative version of the value, without considering the positive/negative sign.
```

Comment:
```suggestion
Absolute values are the non-negative versions of the value without considering the positive/negative sign.
```

Reply:
I'm wondering if this sentence makes more sense below the \```psuedo snippet?

```
Absolute values are the non-negative versions of the value `n` without considering the positive/negative sign.
```